app/code/Magento/AsynchronousOperations/Model/ResourceModel/Operation/OperationRepository.php

```diff
@@ -91,7 +91,7 @@ public function createByTopic($topicName, $entityParams, $groupId, $operationId)
         ];
         $data = [
             'data' => [
-                OperationInterface::ID => $operationId,
```

Comment:
Don't forget this class exists as well in the webapi async module, and needs to be changed there as well.

Magento/WebApiAsync/Model/OperationRepository.php:92